### PR TITLE
Test against 10.3.0alpha

### PR DESCRIPTION
**Pull request by phil-davis**
Issue #358 

10.3.0alpha is available, so use that for now. (later beta/RC/release will come...)

### CODE REVIEW

**Comment by phil-davis** [comment]
@patrickjahns @individual-it 
1) because the alpha tarball is in the `testing` folder under community downloads, I had to specify `download_url` rather than just being able to specify `oc_version`. I guess that is OK, because we do not want these test tarballs to appear in the top-level community download folder (people might "accidentally" install them in production)

2) the tarball name will increment frequently to e.g. `10.3.0beta` `10.3.0RC1` `10.3.0RC2` ... I wonder how we can put something in the app drone yml that says "use the latest testing tarball"?

File: .drone.yml [hunk 1]
```diff
@@ -439,30 +452,29 @@ matrix:
...
-#      DB_NAME: owncloud
-#      NEED_SERVER: true
-#      NEED_INSTALL_APP: true
-#      USE_EMAIL: true
-#      NEED_CORE: true
-#      USE_RELEASE_TARBALL: true
-#      FIX_PERMISSIONS: true
+      # Release Tarball
+    - PHP_VERSION: 7.1
+      DOWNLOAD_URL: https://download.owncloud.org/community/testing/owncloud-10.3.0alpha.tar.bz2
```
Comment (individual-it) [inline .drone.yml]: does `OC_VERSION` not work?

Comment (phil-davis) [reply .drone.yml]: The tarball is in the `community/testing`  folder.
`OC_VERSION` looks in the `community`  folder only.

Comment (patrickjahns) [reply .drone.yml]: Please create a FR in https://github.com/owncloud-ci/core or provide a PR that would search both places before giving up ;-)

Comment (phil-davis) [reply .drone.yml]: PR https://github.com/owncloud-ci/core/pull/19 has been merged.
This PR has been adjusted to remove this special `DOWNLOAD_URL` stuff.

**Pull request merged**